atmo/clusters/jobs.py

```diff
@@ -72,9 +72,11 @@ def update_clusters_info():
     """
     # only update the cluster info for clusters that are pending
     active_clusters = Cluster.objects.active()
+    # get the datetime of the oldest active cluster, set to the start of the day
+    oldest_start_date = active_clusters.datetimes('start_date', 'day')
```

Comment:
According to the docs, `datetimes()` returns a queryset, so `oldest_start_date` will evaluate to a list. You may want  to check that the list has at least one item and eventually pass it to `cluster_list()`.